app/templates/error/401.html

```diff
@@ -1,9 +1,16 @@
 {% extends "document_download_template.html" %}
-{% block per_page_title %}Unauthorized{% endblock %}
+{% block per_page_title %}Sorry, there’s a problem with the service{% endblock %}
```

Comment:
@quis will `– GOV.UK Notify` be automatically appended to the page title?

Reply:
At the moment this line of code does nothing. If you want it to work then it needs to be `page_title` (the Document Download codebase doesn’t have concept of `per_page_title`). It won’t automatically append anything, but I’d suggest that Document Download shouldn’t make reference to Notify, therefore I’d suggest
```suggestion
{% block per_page_title %}Sorry, there’s a problem with the service – GOV.UK{% endblock %}
```

Reply:
You pre-empted my next question 🙂 
Much better that we remove the reference to Notify. I just wanted to check that it wasn't going to auto-append our service name (as it does on /notifications-admin)